moropy_bot/bot.py

```diff
@@ -1,9 +1,11 @@
 # bot.py
 import os
+
 import random
 from discord.ext import commands
 
 import discord
+
 from dotenv import load_dotenv
 
 load_dotenv()
```

Comment:
I think we need to comment this out because there is no env to load.

Reply:
Will it create an error if `.env` is not present? Can you lookup `dotenv` documentation for this?